### PR TITLE
Created modularized navbar and social icon files and fixed indentatio…

### PR DESCRIPTION
**Pull request by AirwreckEye**
…n/spacing.

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
-     </p>
-
-    </div>
-    <!-- Javascript for Bootstrap -->
-  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
-  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
-  </body>
-</html>
+</html>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html:53]: all files should have an empty new line at the bottom

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
+	<!-- Javascript for Bootstrap -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+	<script>
+		loadSelector("#nav", "navbar.html");
+	</script>
+  <script>
+		loadSelector("#social", "socials.html");
+	</script>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/about.html]: You can combine both js method calls into one <script> tag:

```suggestion
	<script>
		loadSelector("#nav", "navbar.html");
		loadSelector("#social", "socials.html");
	</script>
```

Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html]: 👍

File: portfolio/src/main/webapp/gallery.html [hunk 3]
```diff
@@ -1,70 +1,54 @@
...
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<!-- Javascript for Bootstrap -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+	<script>
+		loadSelector("#nav", "navbar.html");
+	</script>
+  <script>
+		loadSelector("#social", "socials.html");
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/gallery.html]: nit: "selector" is kind of ambiguous, maybe "loadHtml"? or something like that

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -1,28 +1,5 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function loadSelector(selector, filename) {
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/script.js]: let's add documentation for the function and params for good measure

see here: https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Comment (codewithjess) [inline portfolio/src/main/webapp/script.js]: Its helpful to add method comments that document what the method does, along with the parameters.
https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (codewithjess) [inline portfolio/src/main/webapp/about.html:53]: Lets add a new line to the end of all the files
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: portfolio/src/main/webapp/about.html [hunk 5]
```diff
@@ -1,48 +1,55 @@
...
-  <head>
-    <meta charset="UTF-8">
-    <title>About Me</title>
-     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
 
-  
+<head>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/about.html:4]: Did you use the prettier formatter on these html files? Not sure why it increased the indentation spacing so much but it should be 2 spaces like before.

**Review by codewithjess — COMMENTED** [review]
Nice job moving the navbar and social icons to separate files.

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (AirwreckEye) [reply portfolio/src/main/webapp/about.html:53]: Noted

**Review by rharagutchi — APPROVED** [review]

File: portfolio/src/main/webapp/about.html [hunk 6]
```diff
@@ -39,17 +37,17 @@ <h3>
...
-  <div class="container">
+	<div class="container">
 		<br>
 		<img src="media/child.JPG" alt="Young tyke" class="img-circle" style="width:410px;height:410px;">
 	</div>
-  <div id="social"></div>
+		<div id="social"></div>
 </body>
 
 </html>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/about.html:53]: nit: looks like this file is missing a new line at the end

File: portfolio/src/main/webapp/topten.html [hunk 7]
```diff
@@ -15,10 +15,8 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
 	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
 	<script>
-		loadSelector("#nav", "navbar.html");
-	</script>
-  <script>
-		loadSelector("#social", "socials.html");
+	  loadHTML("#nav", "navbar.html");
+		loadHTML("#social", "socials.html");
```
Comment (codewithjess) [inline portfolio/src/main/webapp/topten.html]: nit: the indentation is off here. lines 18-19 should be at the same indent level.

**Review by codewithjess — APPROVED** [review]

File: portfolio/pom.xml [hunk 8]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (codewithjess) [inline portfolio/pom.xml:37]: You can remove this TODO comment now :)

**Pull request merged**